docs/installation/registry-mirror.md

```diff
@@ -142,5 +142,29 @@ You would then be able to pull those images from all cluster members, as long as
 The embedded registry is read-only, and cannot be pushed to directly using `docker push` or other common tools that interact with OCI registries.
 
 Images can be manually made available via the embedded registry by running `ctr -n k8s.io image pull` to pull an image,
-or by loading image archives created by `docker save` via the `ctr -n k8s.io image import` command.
+or by loading image archives created by `docker save` via the `ctr -n k8s.io image import` command or the [auto-import feature](#auto-import-images).
 Note that the `k8s.io` namespace must be specified when managing images via `ctr` in order for them to be visible to the kubelet.
+
+## Auto-import images
+
+:::info Version Gate
```

Comment:
Pulling images from a text file has always worked. The bit that's new is being able to drop tarballs or text files into the images for while k3s is running, and they will be pulled/imported. Previously the pull/import only happened on startup. 

This should be clarified, as this makes it sound like it is the text file support that is new.